recipes/avahi/all/test_package/CMakeLists.txt

```diff
@@ -5,3 +5,7 @@ find_package(Avahi CONFIG REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.c)
 target_link_libraries(${PROJECT_NAME} PRIVATE avahi::compat-libdns_sd avahi::core)
+
```

Comment:
Please, revert your test package changes. The idea of test package is validating the package itself, not running full examples, we just want to check for artifacts and linkage status, so we keep as simple as possible always.

I see you are trying to validate this explored bug, so I would suggest only adding:

```
#include <avahi-common/malloc.h>
#include <avahi-common/error.h>
```

to `all/test_package/test_package.cpp`

Reply:
OK, I've updated the package accordingly that both `<malloc.h>` and `<avahi-common/malloc.h>` are included.